src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -58,6 +58,14 @@
  * queue. For example, {@code Robot.mouseMove} will actually move
  * the mouse cursor instead of just generating mouse move events.
  * <p>
+ * Note: {@code waitForIdle()} must not be called on the AWT EDT, since
```

Comment:
The word "and" needs to be before "since" for this to make sense and then that needs more rewording later.

Mulling it over, I suggest rewording like this

<pre>
 * Note: When {@code autoWaitForIdle()} is enabled, mouse and key related methods
 * cannot be called on the AWT EDT.
 * This is because when {@code autoWaitForIdle()} is enabled,
 * the mouse and key methods implicitly call {@code waitForIdle()}
 * which will throw {@code IllegalThreadStateException} when called on the AWT EDT.
</pre>